### PR TITLE
Rename addons to platform services in kommander docs

### PR DESCRIPTION
**Pull request by gracedo**
## Jira Ticket
Before creating this pull request, make sure you have a separate ticket for the docs team to track their work. If you need assistance, ask in the #documentation Slack channel.

https://jira.d2iq.com/browse/D2IQ-73666
docs review ticket: https://jira.d2iq.com/browse/D2IQ-73669

## Description of changes being made
We are renaming "addons" to "platform services" across Kommander UI and docs for kommander 1.3. Konvoy rename will happen later with konvoy/kommander unification work.

## Checklist
- [ ] Change all affected versions, if applicable (e.g. 1.13, 2.0, 2.1).
- [ ] Test all commands and procedures, if applicable.
- [ ] Create your PR against `staging`, not `master`. 
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> ie `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://wiki.d2iq.com/display/ENG/Contributing+to+Docs) for more information.

### CODE REVIEW

**Review by cneth — APPROVED** [review]

**Comment by cmastr** [comment]
@gracedo I'd like to hold approval of this PR until I've seen the development PR that matches. There are direct references to pages that, if the page itself is not changed, will make the docs seem wrong.

**Comment by gracedo** [comment]
> @gracedo I'd like to hold approval of this PR until I've seen the development PR that matches. There are direct references to pages that, if the page itself is not changed, will make the docs seem wrong.

We have a UI PR open that changes the language in the UI, is that the dev pr you are referring to? https://github.com/mesosphere/kommander-ui/pull/3223

**Comment by gracedo** [comment]
@cmastr since this is queued to all go out in kommander 1.3, can we get this merged? I'm working on the feature docs in parallel which build on top of many of these changes so if we merged this in first I could avoid merge conflicts in those branches!

File: pages/dkp/kommander/1.3/clusters/index.md [hunk 1]
```diff
@@ -181,8 +181,8 @@ data:
 ```
 
 | Key | Description | Required |
-| :--- | :--- | :---: | 
-| metadata . labels . "d2iq.io/addon" | The addon name (id) | X |
+| :--- | :--- | :---: |
+| metadata . labels . "d2iq.io/addon" | The platform service name (id) | X |
```
Comment (cmastr) [inline pages/dkp/kommander/1.3/clusters/index.md:185]: @gracedo where we say "platform service name", is that referencing the product UI?

**Comment by cmastr** [comment]
> @cmastr since this is queued to all go out in kommander 1.3, can we get this merged? I'm working on the feature docs in parallel which build on top of many of these changes so if we merged this in first I could avoid merge conflicts in those branches!

In this case, we should merge when the PR you referenced merges. I assume it'll be before Kommander 1.3 GA?

**Comment by gracedo** [comment]
@cmastr yep! And that PR just merged so we should be good to go on this PR :)

File: pages/dkp/kommander/1.3/clusters/attach-cluster/federated-platform-services/index.md [hunk 2]
```diff
@@ -1,36 +1,36 @@
...
 
 On attachment, two factors impact successfully deploying an addon on the attached cluster:
 
 1. Is the attached cluster a Konvoy cluster or not? For example, a cluster deployed using AWS EKS.
 2. Does a label describing the federation of the addon exist and is it defined? If yes, what is its value?
 
-The following tables describe the list of addons and cluster addons that get federated on attachment with their related federation label (if available). Addons that do not have a federation label are federated by default. If the addon description indicates only federated on non-Konvoy clusters, the addon will not get installed into Konvoy clusters, even if its federation label is set to `true`.
+The following tables describe the list of platform services and cluster platform services that get federated on attachment with their related federation label (if available). Platform services that do not have a federation label are federated by default. If the addon description indicates only federated on non-Konvoy clusters, the addon will not get installed into Konvoy clusters, even if its federation label is set to `true`.
```
Comment (msdolbey) [inline pages/dkp/kommander/1.3/clusters/attach-cluster/federated-platform-services/index.md:19]: I don't really know, but should these other instances of platform services be included in this graf?

```suggestion
The following tables describe the list of platform services and cluster platform services that get federated on attachment with their related federation label (if available). Platform services that do not have a federation label are federated by default. If the platform service description indicates only federated on non-Konvoy clusters, the platform service will not get installed into Konvoy clusters, even if its federation label is set to `true`.
```

File: pages/dkp/kommander/1.3/clusters/attach-cluster/federated-platform-services/index.md [hunk 3]
```diff
@@ -1,36 +1,36 @@
...
 
 Currently, the monitoring stack is federated by default and the logging stack is not. This is why `prometheus` is the only addon federated by default with a federation label to disable it if needed.
```
Comment (msdolbey) [inline pages/dkp/kommander/1.3/clusters/attach-cluster/federated-platform-services/index.md:21]: @gracedo, Should this be included, too?

```suggestion
Currently, the monitoring stack is federated by default and the logging stack is not. This is why `prometheus` is the only platform service federated by default with a federation label to disable it if needed.
```

File: pages/dkp/kommander/1.3/clusters/attach-cluster/federated-platform-services/index.md (hunk 2 above)
Comment (gracedo) [reply pages/dkp/kommander/1.3/clusters/attach-cluster/federated-platform-services/index.md:19]: oops yes I missed those

File: pages/dkp/kommander/1.3/clusters/attach-cluster/federated-platform-services/index.md (hunk 3 above)
Comment (gracedo) [reply pages/dkp/kommander/1.3/clusters/attach-cluster/federated-platform-services/index.md:21]: yes! not sure how I missed all these!

File: pages/dkp/kommander/1.3/projects/platform-services/index.md [hunk 4]
```diff
@@ -167,6 +167,6 @@ status:
 
 The Konvoy Addon Controller is then responsible for deploying the application.
 
-After the addon is deployed, the addon may be upgraded to a newer version. The only versions shown are compatible with the clusters in that project.
+After the platform service is deployed, the platform service may be upgraded to a newer version. The only versions shown are compatible with the clusters in that project.
 
-![Project addon edit form](/dkp/kommander/1.3/img/project-catalog-addon-edit.png)
+![Project platform service edit form](/dkp/kommander/1.3/img/project-catalog-addon-edit.png)
```
Comment (msdolbey) [inline pages/dkp/kommander/1.3/projects/platform-services/index.md:172]: Do we have to worry about changing the name of the img files and stuff like that, or are we good? I'm leaning toward this is okay to me. More of a @cmastr / docs management question, I suppose.

Comment (cmastr) [reply pages/dkp/kommander/1.3/projects/platform-services/index.md:172]: no on image files.

**Pull request merged**